cmd/lk/sip.go

```diff
@@ -128,6 +129,12 @@ var (
 							Action:    createSIPParticipant,
 							ArgsUsage: RequestDesc[livekit.CreateSIPParticipantRequest](),
 						},
+						{
+							Name:      "transfer",
+							Usage:     "Transfer a SIP Participant",
+							Action:    transferSIPParticipant,
+							ArgsUsage: RequestDesc[livekit.TransferSIPParticipantRequest](),
```

Comment:
This API is pretty close to the ones we have on room participants in general, so maybe we should not require a JSON file here? Could be all 3 arguments instead: `--room`, `--id` (or how do we call participant identity in CLI?) and regular argument for the transfer URI.

Reply:
++ on using named args. We usually use `--identity` for participants, and could map the rooms to `--from` and `--to`.

Reply:
To be consistent with other commands, should we use `--room`, `--identity` `--to`? It may be confusing to have `from` be a room, and `to` a sip url